packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -79,15 +103,24 @@ export default class FlyoutHelpStore {
 
         const getNextBlock = async (xml, current_index, direction) => {
             const next_index = current_index + (direction ? 1 : -1);
-            const block_type = Object.keys(xml).find((key, index) => next_index === index);
-
-            if (!block_type) {
+            const next_blocks_Type = Object.keys(xml).filter((key, index) =>
```

Comment:
```suggestion
            const next_block_type = Object.keys(xml).filter((key, index) =>
```

Reply:
The return value is an array, so I named it as plural, 4 lines under it I used "next_block_type" for storing one block type.

Reply:
Alright, that's fine, but please use all lower case for variables (unless we're working with functions).